pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
I think rounded results is a better default than non-rounded and not only for this function but other like `df_summary`. AFAIK the only other function with a "round_to" argument is `plot_posterior`. Anyway I am totally open to your suggestions.